lib/rspec/its.rb

```diff
@@ -110,7 +110,8 @@ def its(attribute, *options, &block)
               subject[*attribute]
             end
           else
-            attribute_chain = attribute.to_s.split('.')
+            split_attribute_by = !options.empty? && options[0][:split_attribute_by] ? options[0][:split_attribute_by] : '.'
```

Comment:
`options` should be a hash, the metadata hash, we have tests relying on this so I'm confused why its an array here? Also stylistically we'd do ` options.fetch(:split_attribute_by, '.')` rather than a ternary.